packages/kubectl/packaging

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+
+VERSION=v1.13.2
+
+echo Retrieving kubetcl ${VERSION}
+
+curl -LO https://storage.googleapis.com/kubernetes-release/release/${VERSION}/bin/linux/amd64/kubectl
```

Comment:
this is not ok
you could use `pre_packaging` to do this
the prefered way, however, is to use a blob
I noticed the same pattern in SCF - it's a bug.

example: https://github.com/SUSE/scf/blob/develop/src/scf-release/config/blobs.yml#L6-L9

Reply:
Added kubectl as a blob: https://github.com/cloudfoundry-community/eirini-bosh-release/pull/14/commits/0b821190329a3d7e2c4043b43faae5c2c8b71a21